packages/connect-explorer/src/actions/trezorConnectActions.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-underscore-dangle */
 import TrezorConnect, { DEVICE, DEVICE_EVENT, TRANSPORT_EVENT } from '@trezor/connect-web';
```

Comment:
export { TrezorConnectProxy } ?
maybe { TrezorConnectForegroundProxy } ?

Reply:
if we are building connect-explorer for webextension we should use @trezor/connect-webextension (possibly replace by webpack)